usage/app/model/SyndicationUsageRequest.scala

```diff
@@ -7,12 +7,12 @@ import play.api.libs.json._
 case class SyndicationUsageRequest (
   partnerName: String,
   syndicatedBy: Option[String],
-  startPending: Option[Boolean],
+  startPending: Option[String],
```

Comment:
Still reading through some of the other files, so sorry if I'm missing something, but it does feel like this is a bit more appropriate as a boolean - it looks like we mainly want to be able to say whether `startPending` is true or false?